### DISCUSSION TITLE
It might be better not to use a Creative Commons license.

### DISCUSSION THREAD

**Discussion by jiyu3** [fishaudio/fish-speech#522] [General]
Hi

fish-speeech's license is now CC BY-NC-SA, but actually [Creative Commons does not recommend to use CC license for software](https://creativecommons.org/faq/#can-i-apply-a-creative-commons-license-to-software).

The ShareAlike (SA) rule is particularly problematic. I call the software which is licensed by CC SA "SA-soft".
It's not for sure if the outputs generated by the SA-soft are licensed by CC SA or not.
There is no clear mention of that in the license text because CC doesn't target software works.

So, I strongly recommend to use the license for the software like MIT, GPL and so on.

**Comment by leng-yue** [comment]
Feel free to share the generated content, as long as its not for commercial purpose :)
